pkg/cmd/runner.go

```diff
@@ -76,6 +76,10 @@ func (r *Runner) Run(yaml *config.YAMLConfig) {
 				if err != nil {
 					r.errs = append(r.errs, err)
 					statsMgr.StatsCh <- base.NewFileDoneStats(filename)
+					// fix issues/219
+					// May cause base.FILEDONE to send multiple times, so <-statsMgr.DoneCh return, and then program exit.
+					// It may lead to incomplete import, or even panic.
+					return
```

Comment:
I'm confused about the  purpose of this `return`. Even though without this `return`, this function still always exit when it reaches here.

Reply:
Yes, I am solving. It's wip.